app/Casts/UTCDateTime.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Casts;
+
+use Exception;
+use Illuminate\Contracts\Database\Eloquent\CastsAttributes;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Carbon;
+
+class UTCDateTime implements CastsAttributes
```

Comment:
Can you add a PHPDoc where you explain the usage of this Cast, so we know in future?